### PR TITLE
since the executor is shared y all partition threads the semaphore sh…

### PR DESCRIPTION
**Pull request by Bijnagte**
…ould be as well

@mortent @charliek

### CODE REVIEW

File: src/main/java/smartthings/konsumer/stream/ThreadedMessageConsumer.java [hunk 1]
```diff
@@ -43,18 +43,18 @@
 
 	public ThreadedMessageConsumer(
 			KafkaStream<K, V> stream, Executor messageExecutor, ListenerConfig config,
```
Comment (mortent) [inline src/main/java/smartthings/konsumer/stream/ThreadedMessageConsumer.java]: I don't think the ListenerConfig is used anymore with this change. So we no longer need to pass it in to the MessageConsumer

**Comment by mortent** [comment]
Other than the small comment, this looks good to me. I'm not sure if we are running with more than one partition thread anywhere right now, but this change would be needed for it to work correctly.

**Comment by charliek** [comment]
Same comment as @mortent. Other than that look good to me as well.

**Comment by Bijnagte** [comment]
@mortent and @charliek fixed

**Pull request merged**